### PR TITLE
Release highlights for the week ending December 10

### PR DESCRIPTION
**Pull request by snizhana-dynnyk**
### Please remember to

- [x] Give the PR a meaningful title -- it will be shown in the release notes
- [x] Add (or remove) [user questions](https://github.com/giantswarm/docs/blob/main/CONTRIBUTING.md#front-matter) associated with any updated docs

### CODE REVIEW

**Comment by gianfranco-l** [comment]
@snizhana-dynnyk  functional tests are only for interal use so only the fix is worth mentioning because the problem that solved affected a customer.
so the change would be 

- [flux-app](https://github.com/giantswarm/flux-app) [v0.8.0](https://github.com/giantswarm/flux-app/releases/tag/v0.8.0) includes `helm-controller` fix to reduce memory usage by downgrading `Helm` from `3.7.1` to `3.6.3`.

File: src/content/changes/highlights/2021-12-10.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+title: Highlights for the week ending December 10, 2021
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## Apps
+
+- [flux-app](https://github.com/giantswarm/flux-app) [v0.8.0](https://github.com/giantswarm/flux-app/releases/tag/v0.8.0) adds functional and upgrade tests and includes `helm-controller` fix to reduce memory usage by downgrading `Helm` from `3.7.1` to `3.6.3`.
```
Comment (snizhana-dynnyk) [inline src/content/changes/highlights/2021-12-10.md:12]:
```suggestion
- [flux-app](https://github.com/giantswarm/flux-app) [v0.8.0](https://github.com/giantswarm/flux-app/releases/tag/v0.8.0) includes `helm-controller` fix to reduce memory usage by downgrading `Helm` from `3.7.1` to `3.6.3`.
```

File: src/content/changes/highlights/2021-12-10.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+- [flux-app](https://github.com/giantswarm/flux-app) [v0.8.0](https://github.com/giantswarm/flux-app/releases/tag/v0.8.0) includes `helm-controller` fix to reduce memory usage by downgrading `Helm` from `3.7.1` to `3.6.3`.
+- [dex-app](https://github.com/giantswarm/dex-app) [v1.21.0](https://github.com/giantswarm/dex-app/blob/master/CHANGELOG.md#1210---2021-12-09) contains update to the upstream version `v2.30.2`
+
+## User interfaces
+
+- In the Web UI we now display release status in cluster creation form.
+
+## Dashboards
+
+- Added "Name tbd" dashboard that shows error and success rates for dex running in workload and management clusters.
```
Comment (marians) [inline src/content/changes/highlights/2021-12-10.md:21]: The dashboard publishing won't happen today

File: src/content/changes/highlights/2021-12-10.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+---
+
+## Apps
+
+- [flux-app](https://github.com/giantswarm/flux-app) [v0.8.0](https://github.com/giantswarm/flux-app/releases/tag/v0.8.0) includes `helm-controller` fix to reduce memory usage by downgrading `Helm` from `3.7.1` to `3.6.3`.
+- [dex-app](https://github.com/giantswarm/dex-app) [v1.21.0](https://github.com/giantswarm/dex-app/blob/master/CHANGELOG.md#1210---2021-12-09) contains update to the upstream version `v2.30.2`
+
+## User interfaces
+
+- In the Web UI we now display release status in cluster creation form.
```
Comment (marians) [inline src/content/changes/highlights/2021-12-10.md:17]: I'm thinking this is not really noteworthy today, as the topic of releases will be big next week.

**Comment by gianfranco-l** [comment]
everything good from my side

**Review by gianfranco-l — APPROVED** [review]

**Review by marians — APPROVED** [review]

**Pull request merged**